src/spyglass/lfp/lfp_merge.py

```diff
@@ -0,0 +1,53 @@
+import datajoint as dj
+from spyglass.lfp.v1 import LFP, ImportedLFP
+
+schema = dj.schema("lfp")
+
+
+@schema
+class LFPOutput(dj.Manual):
+    definition = """
+    lfp_id: uuid
+    ---
+    stream: varchar(40)
+    """
+
+    class LFP(dj.Part):
+        definition = """
+        -> LFPOutput
+        ---
+        -> LFP
+        """
+
+    class ImportedLFP(dj.Part):
+        definition = """
+        -> LFPOutput
+        ---
+        -> ImportedLFP
+        """
+
+    @staticmethod
+    def get_lfp_object(key: dict):
```

Comment:
I had added a `fetch_nwb` to each part table so that a user could in theory run `LFPOutput.ImportedLFP.fetch_nwb()` or `.fetch1_dataframe()` in place instead of having to get the keys and go upstream. See [here](https://github.com/LorenFrankLab/spyglass/blob/011dcaaca1f1dc20f1cc75c0edd8f2a85ad48ce4/src/spyglass/position/position_merge.py#LL58C1-L58C1). 
I also added the AnalysisNwbfile as a [secondary](https://github.com/LorenFrankLab/spyglass/blob/011dcaaca1f1dc20f1cc75c0edd8f2a85ad48ce4/src/spyglass/position/position_merge.py#L52) key to each part table that would just carry through the object_id from the upstream table.